docs/resources/azure_migrate_assessment_machine.md

```diff
@@ -0,0 +1,107 @@
+---
+title: About the azure_migrate_assessment_machine Resource
+platform: azure
+---
+
+# azure_migrate_assessment_machine
+
+Use the `azure_migrate_assessment_machine` InSpec audit resource to test properties related to an Azure Migrate Assessment Machine.
+
+## Azure REST API version, endpoint and http client parameters
+
+This resource interacts with api versions supported by the resource provider.
+The `api_version` can be defined as a resource parameter.
+If not provided, the latest version will be used.
+For more information, refer to [`azure_generic_resource`](azure_generic_resource.md).
+
+Unless defined, `azure_cloud` global endpoint, and default values for the http client will be used.
+For more information, refer to the resource pack [README](../../README.md).
+
+## Availability
+
+### Installation
+
+This resource is available in the [InSpec Azure resource pack](https://github.com/inspec/inspec-azure).
+For an example `inspec.yml` file and how to set up your Azure credentials, refer to resource pack [README](../../README.md#Service-Principal).
+
```

Comment:
Please add the link for this resource api.

Reply:
done..